wondrous-app/components/Common/SidebarEntityList/index.tsx

```diff
@@ -86,19 +87,13 @@ const useSidebarData = () => {
             text: 'Collaborations',
             Icon: SmallDao2DaoIcon,
             link: {
-              pathname: router.pathname,
+              pathname: router.pathname?.replace('/pods', '/boards'),
```

Comment:
what is this for? @Lamperoyge  will this break anything?

Reply:
think it might, yeah not sure why we need this? @SankhaChak

Reply:
the back button on the sidebar typically takes the user to the task board, if we don't add this, when the user navigates to the collaborations page from pods page, on clicking back, it would take the user to the pods page